### PR TITLE
bpo-43425: Update setup.py not to use distutils.log

### PR DESCRIPTION
**Pull request by corona10**
https://bugs.python.org/issue43425

### CODE REVIEW

**Comment by corona10** [comment]
@vstinner 

`renaming build/scripts-3.11/idle3 to build/scripts-3.11/idle3.11` contains in names
Do you have any ideas?
https://github.com/python/cpython/blob/139de04518bd98a975b7c98ab8a38e570dc585e4/Tools/scripts/generate_stdlib_module_names.py#L140

**Comment by corona10** [comment]
`````
{.... '_datetime', 'renaming build/scripts-3.11/idle3 to build/scripts-3.11/idle3.11', '_tracemalloc', ....}
`````

**Review by mmdbalkhi — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -2667,7 +2673,7 @@ def copy_scripts(self):
                 newfilename = filename + fullversion
             else:
                 newfilename = filename + minoronly
-            log.info('renaming %s to %s', filename, newfilename)
+            log.info(f'renaming {filename} to {newfilename}')
```
Comment (vstinner) [inline setup.py:2677]: With your change, I no longer see this message. IMO you should mimick distutils setup() function which sets the logging thredshold to INFO: 

```
  File "/home/vstinner/python/main/./setup.py", line 2698, in main
    setup(# PyPI Metadata (PEP 301)
  File "/home/vstinner/python/main/Lib/distutils/core.py", line 134, in setup
    ok = dist.parse_command_line()
  File "/home/vstinner/python/main/Lib/distutils/dist.py", line 477, in parse_command_line
    log.set_verbosity(self.verbose)
  File "/home/vstinner/python/main/Lib/distutils/log.py", line 78, in set_verbosity
    set_threshold(INFO)
```

To see the message on the main branch:

```
$ rm -rf build/scripts-3.11/
$ make
(...)
renaming build/scripts-3.11/pydoc3 to build/scripts-3.11/pydoc3.11
renaming build/scripts-3.11/idle3 to build/scripts-3.11/idle3.11
renaming build/scripts-3.11/2to3 to build/scripts-3.11/2to3-3.11
```

Comment (corona10) [reply setup.py:2677]: Thank's nice catch :)

**Review by vstinner — APPROVED** [review]
LGTM.

Ok, this time I can ready "renaming ..." logs ;-)

**Pull request merged**